components/feature/sitepermissions/src/main/res/values/strings.xml

```diff
@@ -28,7 +28,7 @@
 
     <!-- Option in a dialog for requesting a camera permission, this option will give access to
      front facing camera-->
-    <string name="mozac_feature_sitepermissions_selfie_camera">Selfie back camera</string>
+    <string name="mozac_feature_sitepermissions_selfie_camera">Selfie camera</string>
```

Comment:
@Pike @Delphine @flodolo What's your recommendation here? Should we update the string id when doing such a change?

Reply:
I wish this was a no, but apparently it's a yes, https://pontoon.mozilla.org/de/android-l10n/all-resources/?search=camera&string=196924.

Reply:
@pocmo should I update the string id? :(

Reply:
Yes, let's do it. :)

Reply:
👍

Reply:
@pocmo  I'm renaming the `mozac_feature_sitepermissions_selfie_camera` to    `mozac_feature_sitepermissions_front_facing_camera`. Should I delete the old strings?

![image](https://user-images.githubusercontent.com/773158/54926879-92ded600-4ee7-11e9-819d-6ae3d161184d.png)